styles/style.css

```
@@ -0,0 +1,276 @@
+  font-weight: 700;
+  color: inherit;
+  text-decoration: none;
+  text-align: center;
+}
+
+.button {
+  font-family: inherit;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  color: #fff;
+  background-color: #756257;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+/* Main */
+
+.main-index {
+  text-align: center;
+}
+
+.main-catalog {
+  background-color: #fff;
+}
+
+/* Wellcome*/
+
+.wellcome {
+  line-height: 36px;
+  background-color: #fff;
+}
+
+.wellcome-banner {
+  background-color: #F2F2F2;
+  background-image: url("../images/background_photo/index-background-1.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+}
+
+.wellcome-title {
+  font-size: 30px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+/* Advantages */
+/* Здесь было сложно с элементами списка. Было два варианта:
+1) Именить список на ненумерованный и включить в него элементы подсписка. Но так страдает семантика;
+2)(К я сделал) Добавить много доп.клоссов и div для подсписка. Но как будто много лишнего кода.
+*3) Мб стоит сделать ненумерованный список, прописать нумерацию текстом и оставить div для подсписка? */
+
+.advantage-item {
+  font-size: 18px;
+  line-height: 21px;
+  background-color: rgba(131, 179, 211, 0.12);
+}
+
+.blue-item {
+  background-color: rgb(131, 179, 211);
+  color: #fff;
+}
+
+.road-item {
+  background-color: rgba(131, 179, 211, 0.2);
+}
+
+.advantage-title {
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 28px;
+  text-transform: uppercase;
+}
+
+/* Advice */
+.advice {
+  color: #000;
+  background-color: #fff;
+}
+
+.eat-item {
+  background-color: inherit;
+}
+
+/* Hotel-searching */
+.hotel-searching {
+  background-color: #fff;
+}
+
+.searching-title {
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 36px;
+  text-transform: uppercase;
+}
+
+.button-search {
+  font-size: 20px;
+  line-height: 36px;
+}
+
+/* Newsletter */
+
+.newsletter {
+  color: #fff;
+  background-color: rgb(131, 179, 211);
+  background-image: url("../images/background_photo/index-background-newsletter.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+}
+
+.newsletter-catalog {
+  color: #000;
+  background-color: #fff;
+  text-align: center;
+}
+
+.newsletter-title {
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 36px;
+  text-transform: uppercase;
+}
+
+.button-blue {
+  font-family: inherit;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  color: #fff;
+  background-color: #83B3D3;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+/* Footer */
+
+.page-footer {
+  font-size: 40px;
+  line-height: 40px;
+  color: #000;
+  background-color: #fff;
+}
+
+.page-footer a {
+  color: #000;
+  background-color: #fff;
+  text-decoration: none;
+}
+
+/* Filter */
+
+.header-filter {
+  color: #fff;
+  background-color: #83B3D3;
+  background-image: url("../images/background_photo/catalog-background 1.jpg");
+  background-size: 100% auto;
```

Да можно сразу. Потом отступы подправят под макет
```suggestion
  background-size: cover;
  background-position: center;
```